Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,17 @@
+class ContactsPhonetic < Formula
+  version "0.2"
+  desc "Add phonetic names to contacts."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/#{version}.tar.gz"
+  sha256 "c6a5deccc7e8f20f89d96f9fde86aa4ae04dba5d7492dec680b839e2fbdfb247"
+
```

Comment:
`contacts-phonetic` depends on `El Capitan` or later?
If so, you should show what OS X version is needed in this formula like this:

```ruby
depends_on :macos => :el_capitan
```

Because Travis CI's mavericks and Yosemite VMs say build failure: https://travis-ci.org/Homebrew/homebrew/builds/106663465